PowerShell.Common.props

```diff
@@ -0,0 +1,23 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Product>PowerShell Core</Product>
+    <Company>Microsoft Corporation</Company>
+    <Copyright>(c) Microsoft Corporation. All rights reserved.</Copyright>
+
+    <FileVersion>6.0.0.0</FileVersion>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
+    <NeutralLanguage>en-US</NeutralLanguage>
```

Comment:
I think we need the `<VersionPrefix>6.0.0</VersionPrefix>` to represent the project version, so that the NuGet package versions are correct.